pkg/templates/nstemplatetiers/nstemplatetier_generator_test.go

```diff
@@ -45,25 +45,31 @@ func TestCreateOrUpdateResources(t *testing.T) {
 				}
 				return clt.Client.Create(ctx, obj)
 			}
-			// verify that no "advanced" nor "basic" NSTemplateTier resources exist prior to creation
-			for _, tierName := range []string{"advanced", "basic"} {
+			// verify that no "advanced" nor "basic" nor "team" NSTemplateTier resources exist prior to creation
+			for _, tierName := range []string{"advanced", "basic", "team"} {
 				tier := toolchainv1alpha1.NSTemplateTier{}
 				err = clt.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: tierName}, &tier)
 				require.Error(t, err)
 				assert.IsType(t, metav1.StatusReasonNotFound, apierrors.ReasonForError(err))
 			}
+			assets := nstemplatetiers.NewAssets(testnstemplatetiers.AssetNames, testnstemplatetiers.Asset)
 
 			// when
-			err := nstemplatetiers.CreateOrUpdateResources(s, clt, namespace, testnstemplatetiers.Asset)
+			err := nstemplatetiers.CreateOrUpdateResources(s, clt, namespace, assets)
 
 			// then
 			require.NoError(t, err)
-			// verify that 2 NStemplateTier CRs were created: "advanced" and "basic"
-			for _, tierName := range []string{"advanced", "basic"} {
+			// verify that 3 NSTemplateTier CRs were created: "advanced", "basic", and "team"
+			for _, tierName := range []string{"advanced", "basic", "team"} {
 				tier := toolchainv1alpha1.NSTemplateTier{}
 				err = clt.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: tierName}, &tier)
 				require.NoError(t, err)
 				assert.Equal(t, int64(1), tier.ObjectMeta.Generation)
+				if tier.Name == "team" {
+					assert.Nil(t, tier.Spec.ClusterResources) // "team" tier should not have cluster resources set
+				} else {
+					assert.NotNil(t, tier.Spec.ClusterResources)
+				}
```

Comment:
it looks a bit weird to me that we have team tier in prod with the ClusterResources set, but here we are missing it. At first glance, it was very confusing. Wouldn't it be better to use a different name for this tier? eg  `nocluster`? 
It's just my point of view - nothing significant.

Reply:
good point. I will add a `cluster.yaml` file in the `team` tier in the `test/templates/nstemplatetiers` pkg, and an extra one called `nocluster` to avoid this confusion

Reply:
done in 780068f